.github/workflows/build-and-deploy-preprod.yml

```diff
@@ -9,6 +9,8 @@ jobs:
   deploy-to-dev:
     name: Bygg app/image, push til github, deploy til dev-fss
     runs-on: ubuntu-latest
+    permissions:
+      packages: write
```

Comment:
Denne må sikkert inn i `build-and-deploy-prod` også?

Reply:
Den er lagt inn der :)